### DISCUSSION TITLE
Error in addGeneExpressionMatrix in ArchR 1.0.3

### DISCUSSION THREAD

**Discussion by rlawltn9718** [GreenleafLab/ArchR#2039] [Questions / Documentation]
I was following the tutorial when I got the error in addGeneExpressionMatrix.

-> Error in addGeneExpressionMatrix(input = projMulti2, seRNA = seRNA_tmp, : Detected less than 50% of seqnames for seRNA in chromSizes! Are you sure the seqnames of your seRNA are correct?!

[ArchR-addGeneExpressionMatrix-13c1b6c06a991-Date-2023-10-23_Time-09-33-22.686473.log](https://github.com/GreenleafLab/ArchR/files/13110982/ArchR-addGeneExpressionMatrix-13c1b6c06a991-Date-2023-10-23_Time-09-33-22.686473.log)

**Comment by sylestiel** [comment]
@rlawltn9718 
Were you able to address this issue?  If so, can you share your solution?

**Comment by ZennFangZhou** [comment]
where to download the 1.0.3 version of ArchR ? i cant find that, thanks!

**Comment by sylestiel** [comment]
try
devtools::install_github("GreenleafLab/ArchR", ref="dev", repos = BiocManager::repositories(), force=TRUE)

**Reply by ZennFangZhou** [reply]
right, thank you!
